rfc/0052-sdk3-full-text-search.md

```diff
@@ -635,6 +635,7 @@ Important SDK considerations:
   * The `k` parameter is mandatory.  The FTS team suggest 3 is a sensible default to send from the SDKs.
 * There are new FTS index definition fields for defining the vector index, but they are inside the `plan` top-level field.  So there is nothing extra required from the SDKs to support these, as we already simply turn the untyped `plan` parameters into JSON.
 * The SDK (and Couchbase), at least in this first phase, have nothing to do with generating the vectors - both those stored in the documents, and those used for queries.  The user will be responsible for generating these, for example using the OpenAI Embeddings API.
+* The feature works unchanged with both global and scoped FTS indexes.
```

Comment:
A few lines above this, it looks like a comment like this:

> (We believe: FTS PM to confirm.)

Was included in a previous commit which added Vector search.  I think we should probably get the answer to that and include the confirmed answer in these changes?

Reply:
Yes that's in progress.  I've put up this very early draft just for initial feedback from the SDK PMs - it's not yet ready for review by the wider team.